media/js/firefox/welcome/welcome8.js

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    function handleOpenProtectionReport(e) {
+        e.preventDefault();
+        Mozilla.UITour.showProtectionReport();
+
+        window.dataLayer.push({
+            'event': 'in-page-interaction',
+            'eAction': 'link click',
+            'eLabel': 'View your protection report'
+        });
+    }
+
+    Mozilla.UITour.ping(function() {
+        document.querySelectorAll('.protection-report').forEach(
```

Comment:
Using `forEach` on a node list may not work in older Firefoxes, but I think here it should be fine, assuming this page is only being shown to recent Firefox versions 👍